packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.ts

```diff
@@ -10,6 +10,7 @@ import {
   SecurityContext,
   TemplateRef,
   ViewChild,
+  ChangeDetectorRef,
```

Comment:
Have you tried using the `OnChanges` lifecycle hook instead of the `ChangeDetectorRef`? https://angular.dev/api/core/OnChanges#